test/e2e/storage/testsuites/snapshottable.go

```diff
@@ -269,9 +269,17 @@ func (s *snapshottableTestSuite) DefineTests(driver storageframework.TestDriver,
 				framework.ExpectNoError(err)
 
 				ginkgo.By("should delete the VolumeSnapshotContent according to its deletion policy")
-				err = storageutils.DeleteAndWaitSnapshot(dc, vs.GetNamespace(), vs.GetName(), framework.Poll, f.Timeouts.SnapshotDelete)
+
+				// Delete both Snapshot and PVC at the same time because different storage systems
```

Comment:
nit: Can this comment be clarified? Does this mean the storage system expects the PVC to be deleted before the snapshot is deleted?

Reply:
Done